### ISSUE TITLE
Supplying seeds on the command line is not safe

### ISSUE THREAD

**Issue by ericvrp** [peterwilli/IOTA-Snapshot-Recovery#1]
Not safe because they end up unencrypted in your command line history file.

It would be better to have then input by the script while running.

**Comment by peterwilli** [comment]
Correct, a simple `read` command should suffice. I'll see what I can do. I'll keep this open until the issue is solved.

**Comment by abhiraj136** [comment]
It is not working for seed not having 81 character.. what to do?

**Comment by peterwilli** [comment]
@abhiraj136 someone forked me and made a better version, I'm going to contact this guy to make a PR so we can improve that ourselves 👍 would be a better collaboration :)

Edit: https://github.com/peterwilli/IOTA-Snapshot-Recovery/issues/3

**Comment by rajivshah3** [comment]
@abhiraj136 if your seed is less than 81, add a 9 until you get to 81 characters (e.g. if your seed is 79, add 2 9's). If your seed is more than 81, enter the first 81 characters
